jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/BufferedResponseHandler.java

```diff
@@ -56,12 +55,14 @@
 public class BufferedResponseHandler extends Handler.Wrapper
```

Comment:
I would like to see most of the impl moved out of this class into a `Content.Sink.BufferedSink` class that can be used by this class.    I think that avoiding blanket buffering (done by a handler) will be important for any app that has a mix of response generation (e.g. anything that has static content does not need extra buffering).

Reply:
Done!